src/components/Footer.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { TodoFilter } from './TodoFilter';
+import { Todo } from '../types/Todo';
+import { FilterValue } from '../types/FiterValue';
+
+type Props = {
+  filterValue: FilterValue;
+  setFilterValue: (filterValue: FilterValue) => void;
+  todos: Todo[];
+  clearCompletedTodos: () => void;
+  completedTodos: Todo[];
+};
+
+export const Footer: React.FC<Props> = ({
+  setFilterValue, filterValue, todos, clearCompletedTodos, completedTodos,
+}) => {
+  const activeTodos = todos.filter(todo => !todo.completed);
+
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {activeTodos.length}
+        {' '}
+        items left
```

Comment:
use template string here
```suggestion
        {`${activeTodos.length} items left`}
```
check other similar cases